XamarinCommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.tizen.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using ElmSharp;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+using Xamarin.Forms.Platform.Tizen;
+using Point = Xamarin.Forms.Point;
+
+[assembly: ExportRenderer(typeof(DrawingView), typeof(DrawingViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+    public class DrawingViewRenderer : ViewRenderer<DrawingView, SKCanvasView>
+    {
+        static EvasObject parent;
+        SKCanvasView canvasView;
+        bool isDrawing;
+        List<Point> points = new List<Point>();
+
+        public static void Init(EvasObject parent) => DrawingViewRenderer.parent = parent;
```

Comment:
You can get parent object by using `Forms.NativeParent` anywhere. So, I don't think we have additional Init() method to get parent object here.